network/rest_interface.c

```diff
@@ -281,7 +281,7 @@ bool construct_rest_header(rest_ctx_t *rest_ctx, char *g_URL,
 		goto err;
 	}
 
-	if (snprintf_s_i(temp, sizeof(temp), "/fido/%d", rest_ctx->prot_ver) <
+	if (snprintf_s_i(temp, sizeof(temp), "/fdo/%d", rest_ctx->prot_ver) <
```

Comment:
PRI is still "/fido" but I assume this is forward looking.

Reply:
Yes. This PR has been run against PR# 144 [https://github.com/secure-device-onboard/pri-fidoiot/pull/144], because it has the updates for protocol implementation differences